test/attr/open.swift

```diff
@@ -134,3 +134,31 @@ open class OpenSubClass : OpenSuperClass {
   open override subscript(index: MarkerForNonOpenSubscripts) -> Int { return 0 }
   
 }
+
+class InvalidOpenExtensionClass { }
+
+open extension InvalidOpenExtensionClass {  // expected-error {{extensions cannot be marked 'open'; mark individual members as 'open' instead}} {{1-6=}} {{3-3=public }} {{9-9=open }} {{9-9=open }} {{3-3=public }}
+  private func A() { } // OK
+  @objc func B() { } // Insert open 
+  func C() { } // Insert public
+  @objc private var D: Int { 3 } // Okay
+  @objc var E: Int { 3 } // Insert open
+  var F: Int { 3 } // Insert public
+  private var G: Int { 3 } // Okay
+}
+
+import Foundation
```

Comment:
Due to use of this `import Foundation`, you should use `-sdk %clang-importer-sdk` in the third `RUN` line, since we don't need the "real" `Foundation`, the fake one in the test suite should be good enough.

Reply:
I didn't know that existed, thanks :) Added and pushed!